docs/xml-config-file.rst

```diff
@@ -40,7 +41,10 @@ Types section is optional.
 Topic Types
```

Comment:
Remove "Topic Types". It can be confuse. Join with section "Types" and defined `<types>` as fastrtps dynamic types and `<is_types>` as types defined through IS.

Reply:
Updated.